chord_metadata_service/chord/workflows/metadata.py

```diff
@@ -147,6 +149,31 @@ def json_file_output(id_: str, output_name: Optional[str] = None):
                 }
             ]
         },
+        WORKFLOW_DOCUMENT: {
+            "name": "Document",
+            "description": "This workflow ingests into DRS documents which "
+                           "have been already listed as experiment results. ",
+            "data_type": DATA_TYPE_EXPERIMENT_RESULT,
+            "file": "document.wdl",
+            "inputs": [
+                {
+                    "id": "document_files",
+                    "type": "file[]",
+                    "required": True,
+                    "extensions": [".pdf", ".csv", ".tsv", ".txt", ".doc", ".docx", ".xls", ".xlsx",
+                                   ".jpeg", ".jpg", ".png", ".gif", ".md", ".mp3", ".m4a", ".mp4"]
+                },
+                DRS_URL_INPUT,
+            ],
+            "outputs": [
+                {
+                    "id": "document_files_out",
+                    "type": "file",
+                    "value": "{document_files_out}"
+                },
```

Comment:
in this case, there shouldn't be any outputs defined here - WES ignores these anyway currently

Reply:
(this part of workflow definitions is going away in the future)